utils/crates-io/Cargo.toml

```
@@ -3,14 +3,11 @@ name = "crates-io-manager"
 version.workspace = true
 edition.workspace = true
 
-[[bin]]
-name = "publish"
-path = "publish.rs"
-
 [dependencies]
 anyhow.workspace = true
 cargo_metadata.workspace = true
 cargo_toml.workspace = true
-crates-io.workspace = true
-curl.workspace = true
+clap = { workspace = true, features = [ "derive" ] }
```

See https://github.com/gear-tech/gear/pull/3542#discussion_r1411068464

```suggestion
clap = { workspace = true, features = ["derive"] }
```